assets/app/views/browse/persistent-volume-claim.html

```diff
@@ -0,0 +1,51 @@
+<div class="content">
+  <project-page>
+    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+    <alerts alerts="alerts"></alerts>
+    <div ng-if="!loaded">Loading...</div>
+    <div ng-if="pvc">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="tile">
+            <h1>
+              {{pvc.metadata.name}}
+              <small class="meta" ng-if="!pvc.spec.volumeName">
+                  <span ng-if="pvc.spec.resources.requests['storage']">
+                    waiting for {{pvc.spec.resources.requests['storage'] | usageWithUnits: 'storage'}} allocation,
+                  </span>
+                  <span ng-if="!pvc.spec.resources.requests['storage']">waiting for allocation,</span>
+              </small>
+              <small class="meta">created <relative-timestamp timestamp="pvc.metadata.creationTimestamp"></relative-timestamp></small>
+            </h1>
+            <div class="resource-details">
+              <dl class="dl-horizontal left">
+                <dt>Status:</dt>
+                <dd>
+                  <status-icon status="pvc.status.phase" disable-animation></status-icon>
+                  {{pvc.status.phase}}
+                  <span ng-if="pvc.spec.volumeName">to volume <strong>{{pvc.spec.volumeName}}</strong></span>
+                </dd>
+                <dt ng-if="pvc.spec.volumeName">Capacity:</dt>
+                <dd ng-if="pvc.spec.volumeName">
+                  <span ng-if="pvc.status.capacity['storage']">
+                    allocated {{pvc.status.capacity['storage'] | usageWithUnits: 'storage'}}
+                  </span>
+                  <span ng-if="!pvc.status.capacity['storage']">allocated unknown size</span>
+                </dd>
+                <dt>Requested Capacity:</dt>
+                <dd>
+                  <span ng-if="pvc.spec.resources.requests['storage']">
+                    {{pvc.spec.resources.requests['storage'] | usageWithUnits: 'storage'}}
+                  </span>
+                  <span ng-if="!pvc.spec.resources.requests['storage']">unknown</span>
```

Comment:
I think this should be `<em>none</em>` instead of unknown, unknown to me implies that we failed to determine what the request was, whereas none implies that one was never set